app/directors-corner.tsx

```diff
@@ -1,5 +1,34 @@
-import WorkInProgress from '@/components/work-in-progress';
+import {DM_Serif_Display, DM_Sans} from "next/font/google";
```

Comment:
Looks like you haven't run a formatter? Rebase onto master as the latest commit adds the formatting and linting hooks.